client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -184,14 +185,23 @@ private ClientConnection waitForConnection() throws ConnectionFailedException, S
                 throw new RuntimeException(e);
             }
         }
+        
+        private void setEventNumber(long minEventNumber) {
```

Comment:
It is a minor question, but I'm wondering why the parameter is prefixed with `min`.